packages/client/src/router/index.js

```diff
@@ -92,7 +92,7 @@ export const routes = [
         name: 'myGrants',
         component: () => import('../views/MyGrantsView.vue'),
         meta: {
-          tabNames: ['interested', 'assigned', 'not-applying', 'applied'],
+          tabNames: [shareTerminologyEnabled() ? 'shared-with-your-team' : 'assigned', 'interested', 'not-applying', 'applied'],
```

Comment:
I'm seeing two minor issues here that I think we could improve with a small refactor. The issues: 

1. Now that we're including a ternary in the `tabNames` list here, it's feeling a bit more complex than would be ideal to inline in the meta definition here, as it's getting a bit hard to read. 
2. We really want the `/my-grants` route above to redirect to the first tab (as opposed to explicitly the interested tab. 

I think we can refactor this to solve both issues pretty neatly by determining the list of tab names elsewhere, like at the top of the file here. (Ideally, we might someday split up route definitions into multiple files and then we could keep this closer to the routes themselves. But for now, top of this file feels fine.) So...

```js
const myGrantsTabs = [
  shareTerminologyEnabled() ? 'shared-with-your-team' : 'assigned', 
  'interested', 
  'not-applying', 
  'applied',
];
```

And then the `/my-grants` route can redirect based on the first tab: `redirect: { name: 'myGrants', params: { tab: myGrantsTabs[0] } }`. And the tabNames here can simply reference `myGrantsTabs`.